dev/io.openliberty.microprofile.config.2.0.internal_fat/fat/src/io/openliberty/microprofile/config/internal_fat/Config20NoCDITests.java

```diff
@@ -55,7 +55,7 @@ public static void setUp() throws Exception {
                         .addProperty("aKey", "aValue");
         WebArchive noCDIWar = ShrinkWrap.create(WebArchive.class, NO_CDI_APP_NAME + ".war")
                         .addPackages(true, NoCDITestServlet.class.getPackage())
-                        .addAsResource(noCDIConfigSource, "META-INF/microprofile-config.properties");
+                        .addAsManifestResource(noCDIConfigSource, "microprofile-config.properties");
```

Comment:
This change is probably not something we want, the spec says it needs to be loaded as a resource. `addAsResource()` will ensure this is possible by putting it in `/WEB-INF/classes/META-INF` whereas `addAsManifestResource` will just put it in the `/META-INF` directory in the root of the .war which is not loadable by the classloader.

I think our implementation will actually read it from either location, but the spec requirement is that it's loadable as a resource.